frontend/src/lib/components/PropertyFilters/PropertyFilters.js

```diff
@@ -29,7 +29,14 @@ const FilterRow = React.memo(function FilterRow({
     }
 
     return (
-        <Row align="middle" className="mt-05 mb-05" data-attr={'property-filter-' + index}>
+        <Row
```

Comment:
Great improvement here! Still looks weird in small screens, but not a blocker for now, it seems unlikely someone would want to update this on a mobile device.

<img width="447" alt="" src="https://user-images.githubusercontent.com/5864173/113622554-02791700-9612-11eb-8b70-1bcfb99655a4.png">